lms/static/sass/_build-lms-v1.scss

```diff
@@ -62,20 +62,7 @@
 @import 'views/api-access';
 
 // app - discussion
-@import "discussion/utilities/variables-v1";
-@import "discussion/mixins";
-@import 'discussion/discussion'; // Process old file after definitions but before everything else, partial is deprecated.
-@import 'discussion/discussion-v1'; // Non-Pattern Library styling
-@import "discussion/elements/actions";
-@import "discussion/elements/editor";
-@import "discussion/elements/labels";
-@import "discussion/elements/navigation";
-@import "discussion/views/home";
-@import "discussion/views/thread";
-@import "discussion/views/create-edit-post";
-@import "discussion/views/response";
-@import 'discussion/utilities/developer';
-@import 'discussion/utilities/shame';
+@import 'build-discussion';
```

Comment:
@andy-armstrong Yes, this was here to make the team's styling work. I had removed discussion-related code from this file in a previous iteration to keep the rules from getting included twice, but that was before you pointed out the "Teams" feature :)

Implemented your suggestion in https://github.com/edx/edx-platform/pull/13483/commits/b50a7dacf9b2da92e832b50f90500a5cc94eccdf.